lib/terminal-table/table.rb

```diff
@@ -155,6 +155,11 @@ def title=(title)
       recalc_column_widths Row.new(self, [title_cell_options])
     end
 
+    # a table is empty if its rows have no useful content (separators aside)
+    def empty?
+      rows.empty?
```

Comment:
In that case, I don't think I will merge this.

I'm not one for programming "laws", but many forms of the "law" of Demeter allow calling methods on objects that the object creates/instantiates, or are direct components. Both of those cases are true here.